backend-lib/src/main/java/cash/z/ecc/android/sdk/internal/model/TorWalletClient.kt

```diff
@@ -65,6 +66,25 @@ class TorWalletClient private constructor(
             TreeStateUnsafe.new(Service.TreeState.parseFrom(treeState))
         }
 
+    suspend fun updateTransparentAddressTransactions(
+        backend: RustBackend,
+        address: String,
+        startHeight: BlockHeightUnsafe?,
+        endHeight: BlockHeightUnsafe?,
+    ) =
+        backend.withWallet { dataDbFile, networkId ->
+            execute {
```

Comment:
This line fails to compile with the error "Suspension functions can be called only within coroutine body". I know conceptually what change needs to be made to `RustBackend.withWallet`, but I have no idea how to do it in Kotlin.

Reply:
Fixed in 3226e9bca2df675124c868e2b4fe26e74ea593c9.